impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/customPseudoScope/PrototypeBean.java

```diff
@@ -0,0 +1,12 @@
+package org.jboss.cdi.tck.tests.build.compatible.extensions.customPseudoScope;
+
+import java.util.UUID;
+
+@Prototype
```

Comment:
This isn't bean defining annotation, you'll need to add a stereotype to this class or have a stereotype that already contains `@Prototype`.
See also `org.jboss.cdi.tck.tests.full.deployment.discovery.BeanDiscoveryTest` and its `MyStereotype` class which does the exact same to register custom pseudo scope.

Reply:
I forgot that pseudo-scope annotations are not BDAs :-/ Will add a stereotype.

Reply:
Done.